lib/java-extras/src/main/java/org/triplea/java/StringUtils.java

```diff
@@ -73,4 +77,24 @@ public static String truncateEnding(
         ? stringToTruncate.substring(0, stringToTruncate.indexOf(endingToTruncate))
         : stringToTruncate;
   }
+
+  /**
+   * Reads the entire contents of an inputStream into a string. The inputStream is *not* closed by
+   * this method, the caller is responsible for closing the input stream.
+   */
+  public static String readFully(final InputStream inputStream) {
+    try {
+      return IOUtils.toString(inputStream, StandardCharsets.UTF_8);
```

Comment:
```suggestion
      return new String(inputStream.readAllBytes(), StandardCharsets.UTF_8);
```

This avoids the need for commons-io, should be the exact same behaviour

Reply:
Cool, I agree we should avoid libraries when there is a relatively straight forward JDK equivalent. Updated: c81c844